Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -227,7 +227,7 @@ void updateLayersForInteractionRegions(const RemoteLayerTreeNode& node)
                 if (applyBackgroundColorForDebugging) {
                     [regionLayer setBorderColor:cachedCGColor({ WebCore::SRGBA<float>(1, 0, 0, .2) }).get()];
                     [regionLayer setBorderWidth:6];
-                    [regionLayer setName:@"Occlusion"];
+                    [regionLayer setName:[NSString stringWithFormat:@"Occlusion %llu", region.elementIdentifier.toUInt64()]];
```

Comment:
Is this worth the extra cost per layer (of which there can be many)? Also I wonder if it would be cheaper to use WTF string building stuff instead of format strings.

Reply:
moving this change to a future patch where it'll be more on-topic